### PR TITLE
api: searching issues by phrases.

### PR DESCRIPTION
**Pull request by doflamingo721**
1. implemented searching issues and displaying them on search page.

### CODE REVIEW

File: lib/src/pages/search_bar.dart [hunk 1]
```diff
@@ -1,46 +1,141 @@
...
-            ),
-          ],
-        ),
+              ],
+            )),
       ),
     );
   }
+
+  Future<void> buildSearIssuePage() async {
```
Comment (FaiyazMujawar) [inline lib/src/pages/search_bar.dart]: Here's a typo!

**Review by FaiyazMujawar — CHANGES_REQUESTED** [review]
Everything looks good. Just rectify the typo. Also, a note: if the PR is about search page, you should not include something which is not related to search page. You have included changes to profile page as well. Else everything is ok.

**Review by Siddharaj-Jawalkar — COMMENTED** [review]
looks everything fine to me.

**Comment by doflamingo721** [comment]
Completed all the necessary changes

**Comment by drigio** [comment]
Good work @doflamingo721 @FaiyazMujawar !

**Pull request merged**